serverlessworkflow/modules/ROOT/pages/job-services/core-concepts.adoc

```diff
@@ -1,107 +1,584 @@
+More information on this image can be found link:{jobs_service_image_usage_url}[here].
+
+We recommend that you follow this procedure:
+
+1. Identify the <<job-service-persistence,persistence mechanism>> to use and see the required configuration parameters.
+2. Identify if the <<job-service-eventing-api, Eventing API>> is required for your needs and see the required configuration parameters.
+3. Identify if the project containing your workflows is configured with the appropriate xref:job-services/quarkus-extensions.adoc#job-service-quarkus-extensions[Job Service Quarkus Extension].
+
+Finally, to run the image, you must use the <<exposed-environment-variables, environment variables exposed by the image>>, and other configurations that you can set using <<using-environent-variables, additional environment variables>> or using <<using-java-like-system-properties, system properties with java like names>>.
+
+[#exposed-environment-variables]
+=== Exposed environment variables
+
+[cols="1,2"]
+|===
+|Variable | Description
+
+|`SCRIPT_DEBUG`
+|Enable debug level of the image and its operations.
+
+|`JOBS_SERVICE_PERSISTENCE`
+|Any of the following values:  `postgresql`, `ephemeral`, or `infinispan` to select the persistence mechanism to use, <<job-service-persistence, see>>.
+
+|===
+
+[NOTE]
+====
+If used, these values must always be passed as environment variables.
+====
+
+[#using-environent-variables]
+=== Using environment variables
+
+To configure the image by using environment variables you must pass one environment variable per each parameter.
+
+[source, bash]
+----
+docker run -it -e JOBS_SERVICE_PERSISTENCE=postgresql -e VARIABLE_NAME=value quay.io/kiegroup/kogito-jobs-service-allinone:latest
+----
+
 [NOTE]
 ====
-Data index service and the support for jobs information will be available in future releases.
+This is the recommended approach when you execute the Job Service in kubernetes.
+The timeouts showcase example xref:use-cases/timeout-showcase-example.adoc#execute-quarkus-project-standalone-services[Quarkus Workflow Project with standalone services] contains an example of this configuration, https://github.com/kiegroup/kogito-examples/blob/main/serverless-workflow-examples/serverless-workflow-timeouts-showcase-extended/kubernetes/jobs-service-postgresql.yml#L65[see].
 ====
 
-=== Job Service persistence
+[#using-java-like-system-properties]
+=== Using system properties with java like names
+
+To configure the image by using system properties you must pass one property per parameter, however, in this case, all these properties are passed as part of a single environment with the name `JAVA_OPTIONS`.
+
+[source, bash]
+----
+docker run -it -e JOBS_SERVICE_PERSISTENCE=postgresql -e JAVA_OPTIONS='-Dmy.sys.prop1=value1 -Dmy.sys.prop2=value2' \
+quay.io/kiegroup/kogito-jobs-service-allinone:latest
+----
```

Comment:
Those are docker specific and should be documented as such. Maybe changing the section title ?
Or do not use docker commands as example
Or put some title to the docker commands to explain it is with docker

Anyway, I am not sure we really need to show the command.
Also the `Using system properties with java like names` is just another environment variable to set to the container

I think we could reduce this part.

Reply:
Not sure if I understand your point.
You mean that the option of using   JAVA_OPTIONS='-Dmy.sys.prop1=value1 -Dmy.sys.prop2=value2' is only for docker?
and doesn't work for instance if you define a Kubernets deployment?

Reply:
I meant that the command shown apply only for Docker and should be documented as such. User would ask: What is the configuration for Kubernetes ?

Reply:
ah, ok, I see. Yes, the command example is for docker and In the note we have a link to the examples where the user can see a kubernetes configuration example.

Reply:
@radtriste  a title on the docker example was added, also a kubernetes example fragment with the corresponding title, and in the NOTE we have links to the full example.

I think users will understand: see https://deploy-preview-408--tangerine-crostata-04a6bf.netlify.app/serverlessworkflow/latest/job-services/core-concepts.html#using-environent-variables